api/src/Entity/ContentNode.php

```diff
@@ -92,8 +92,6 @@ abstract class ContentNode extends BaseEntity implements BelongsToContentNodeTre
     /**
      * Holds the actual data of the content node.
      */
```

Comment:
Should we move the comment to the getter as well? This is also displayed in the OpenAPI spec.

Reply:
it is also displayed in the OpenApi spec if the comment is on the field.
https://pr3749.ecamp3.ch/api#model-ContentNode.jsonhal-read

Do you suggest 
to add it to the getter too
or move it to the getter?

Reply:
I would move it to the getter, to make it clear to future devs where all the information in the OpenAPI spec comes from

Reply:
I will try to move it to the getter.

Reply:
done